Added FAQ section detailing how to fix a faulty instance of QBD (#389)
codatio/codat-docs: docs/integrations/accounting/quickbooksdesktop/quickbooks-desktop-troubleshooting.md

:::note Re-linking and company IDs

> The same Company ID must be used when re-completing the linking process.

:::

## The QuickBooks Connector tray icon is red

![Image](/img/integrations/accounting/quickbooksdesktop/unhealthy-qbd-connector-trayicon.png "Unhealthy QuickBooks Connector Icon")

If your QuickBooks Connector tray icon is red, this indicates that the connector application is currently not running. 

> To restore functionality please take the following steps:

Reviewer: ```suggestion
To restore the connector's functionality, do the following:
```

> 1. Open the Windows Task Manager by pressing `Ctrl+Alt+Del` and selecting `Task Manager`. 

Reviewer: ```suggestion
1. Open the Windows Task Manager by pressing `Ctrl+Alt+Del` and selecting **Task Manager**. 
```

> 2. Go to the `Details` tab of your task manager.

Reviewer: ```suggestion
2. Go to the **Details** tab in the Task Manager.
```

![Image](/img/integrations/accounting/quickbooksdesktop/task-manager-details.png)

> 3. Find the process named `QuickbooksDesktopUI.exe` - Right click on this item and click `End Task`.

Reviewer: ```suggestion
3. Find the process named `QuickbooksDesktopUI.exe`, then right-click on this item and click **End Task**.
```

> 1. Open the Windows Task Scheduler. You can find this by opening your start menu and entering `Task Scheduler`.

Reviewer: Please fix the numbering.

> 2. Find the two tasks named `QuickbooksDesktop_{YOUR_CLIENT_NAME}` and `QuickbooksDesktopUi_{YOUR_CLIENT_NAME}`. The two processes listed below are from ou4 internal development build, your client name would replace "Codat Engineering - Integration".

Reviewer: ```suggestion
2. Find the two tasks named `QuickbooksDesktop_{YOUR_CLIENT_NAME}` and `QuickbooksDesktopUi_{YOUR_CLIENT_NAME}`. The two processes shown below are from our internal development build; your client name would replace "Codat Engineering - Integration".
```

![Image](/img/integrations/accounting/quickbooksdesktop/task-scheduler-qbd-tasks.png "Task Manager entries for QuickBooks Connector processes")

> 3. Right click and select run on the two tasks in order:

Reviewer: ```suggestion
3. Right-click and select **Run** on the two tasks in the following order:
```

    1. `QuickbooksDesktop_{YOUR_CLIENT_NAME}`
    2. `QuickbooksDesktopUi_{YOUR_CLIENT_NAME}`

> The tray icon should now be showing white which indicates a healthy connection. If the tray icon does not re-appear or is still red please try restarting the machine. 

Reviewer: ```suggestion
The tray icon should no longer be red, which indicates a healthy connection. If the tray icon does not reappear or is still red, you can try restarting your machine. 
```

![Image](/img/integrations/accounting/quickbooksdesktop/healthy-qbd-connector-trayicon.png "Healthy QuickBooks Connector Icon")

> If the tray icon remains red after a restart, please contact our Support Team.

Reviewer: ```suggestion
If the tray icon remains red after a restart, contact our Support Team.
```
